src/message_api.c

```diff
@@ -1864,7 +1864,7 @@ static PEP_rating decrypt_rating(PEP_STATUS status)
     case PEP_DECRYPTED:
     case PEP_VERIFY_SIGNER_KEY_REVOKED:
     case PEP_DECRYPT_SIGNATURE_DOES_NOT_MATCH:
-        return PEP_rating_unreliable;
```

Comment:
seems like a trivial change. I briefly tried to look up PEP_DECRYPT_SIGNATURE_DOES_NOT_MATCH, and I think the status is set by sequoia, correct?

Reply:
Shouldn't we move this to

```
case PEP_DECRYPTED:
case PEP_VERIFY_SIGNER_KEY_REVOKED:
    return PEP_rating_unreliable;
case PEP_DECRYPT_SIGNATURE_DOES_NOT_MATCH:
    return PEP_rating_mistrust;
```

Reply:
The tickets talks about "invalid signatures". No signature (PEP_DECRYPTED) and form a revoked key (PEP_VERIFY_SIGNER_KEY_REVOKED) are counted as "invalid". Please note that this is only for encrypted mails.